superset-frontend/src/SqlLab/components/QueryTable/index.tsx

```diff
@@ -125,49 +126,52 @@ const QueryTable = ({
     const statusAttributes = {
       success: {
         config: {
-          icon: <Icons.Check iconColor={theme.colors.success.base} />,
+          icon: <Icons.CheckOutlined iconColor={theme.colors.success.base} />,
+          //icon: <Icons.Edit iconSize="xl" />,
           label: t('Success'),
         },
       },
       failed: {
         config: {
-          icon: <Icons.XSmall iconColor={theme.colors.error.base} />,
+          icon: <Icons.CloseOutlined iconColor={theme.colors.error.base} />,
           label: t('Failed'),
         },
       },
       stopped: {
         config: {
-          icon: <Icons.XSmall iconColor={theme.colors.error.base} />,
+          icon: <Icons.CloseOutlined iconColor={theme.colors.error.base} />,
           label: t('Failed'),
         },
       },
       running: {
         config: {
-          icon: <Icons.Running iconColor={theme.colors.primary.base} />,
+          icon: <Icons.LoadingOutlined iconColor={theme.colors.primary.base} />,
```

Comment:
Is this the animated one? AntD has a way to make it spin IIRC.

Reply:
oh right, I should use that one. One issue I had is not all icons show up as the same size and the smaller ones don't seem to respond to the iconSize prop, we should fix that. It's apparent in the storybook